SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -331,6 +332,21 @@ void PreProcessSchemaMap()
 
 bool SpatialGDKGenerateSchema()
 {
+	// compile all changed blueprints
+	TArray<UBlueprint*> ErroredBlueprints;
```

Comment:
Looking at this bit of code, could this all be moved into its own function and then called here?

Reply:
Pulled it out, lmk if it looks better now.

Reply:
Ignore me, talked to Mike and we agreed to take compilation out of the async task completely.